crates/bitwarden-vault/src/sync.rs

```diff
@@ -79,6 +80,7 @@ pub struct DomainResponse {
 
 #[derive(Serialize, Deserialize, Debug, JsonSchema)]
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
+/// Data returned from `/sync`.
```

Comment:
This is false. `SyncResponseModel` from `bitwarden-api-api` is the response from the endpoint. This is an internal model containing the data returned by calling `sync()`.